docs/Reference/Configuration-File.md

```diff
@@ -4,43 +4,45 @@ description: Configuration File
+| `nodeport`                        | Required | ORION_NODEPORT                        | Port on which to listen for Orion nodes                                                   | `8080`                            |
+| `nodenetworkinterface`            | Optional | ORION_NODENETWORKINTERFACE            | Host on which to listen for Orion nodes                                                   | `"127.0.0.1"`                     |
+| `clienturl`                       | Optional | ORION_CLIENTURL                       | URL advertised to Ethereum clients                                                        | `"http://127.0.0.1:8888"`         |
+| `clientport`                      | Optional | ORION_CLIENTPORT                      | Port on which to listen for Ethereum clients                                              | `8888`                            |
+| `clientnetworkinterface`          | Optional | ORION_CLIENTNETWORKINTERFACE          | Host on which to listen for Ethereum clients                                              | `"127.0.0.1"`                     |
+| `workdir`                         | Optional | ORION_WORKDIR                         | Data directory. This property is relevant only if you use file-based storage, such as LevelDB.  | `. (current directory)`           |
+| `othernodes`                      | Optional | ORION_OTHERNODES                      | Bootnodes for Orion network                                                               | `[] (empty list)`                 |
+| `publickeys`                      | Optional | ORION_PUBLICKEYS                      | List of files containing public keys hosted by node                                       | `[] (empty list)`                 |
+| `privatekeys`                     | Optional | ORION_PRIVATEKEYS                     | List of files containing private keys hosted by node (corresponding order to public keys) | `[] (empty list)`                 |
+| `libsodiumpath`                   | Optional | ORION_LIBSODIUMPATH                   | Path to libsodium shared library                                                          | [Dependent on OS](#libsodiumpath) |
+| `alwayssendto`                    | Optional | ORION_ALWAYSSENDTO                    | List of files containing public keys to include as recipients for every transaction       | `[]`                              |
+| `passwords`                       | Optional | ORION_PASSWORDS                       | File containing [passwords](#passwords) to unlock `privatekeys`                           | `Not set`                         |
+| `storage`                         | Optional | ORION_STORAGE                         | [Storage](#storage) for payloads and related information. Relational databases [PostgreSQL](../Tutorials/Using-PostgreSQL.md) and [Oracle](../Tutorials/Using-Oracle.md) are supported. | `"leveldb"`                       |
```

Comment:
I can't see `knownnodestorage` in the config options?

Reply:
Sorry, added.